cni/pkg/nodeagent/informers.go

```diff
@@ -103,6 +104,25 @@ func (s *InformerHandlers) GetPodIfAmbient(podName, podNamespace string) (*corev
 	return nil, nil
 }
 
+// GetPodIfAmbientEnabled looks up a pod. It returns:
```

Comment:
typo active

And what is the diff with GetPodIfAmbientEnabled

Reply:
Will add better comments generally, this is a copypaste.